### ISSUE TITLE
[Request] Option to change terminal text color

### ISSUE THREAD

**Issue by ajsglist** [AndreyMrovol/LethalTerminalFormatter#18]
Hi there,

I've been looking around to see if there are mods that allow recoloring the terminal text.  I don't believe there are any, and was curious if this was something that could be considered as a feature for terminal formatter?

Thanks

**Comment by AndreyMrovol** [comment]
Hey, it's planned for the future release 😊

**Comment by NakiriFox** [comment]
https://thunderstore.io/c/lethal-company/p/darmuh/darmuhsTerminalStuff/

**Comment by ajsglist** [comment]
Thanks for the suggestion, but I try to avoid mods that are that bloated.  I've seen a number of conflicts and issues reported from that mod.

**Comment by NakiriFox** [comment]
I agree with your point of view, so I have turned off most of the features and only used it as a client-side mod.

**Comment by NakiriFox** [comment]
![image](https://github.com/user-attachments/assets/4be556bd-7bae-4c01-8541-d90913a81718)
![image](https://github.com/user-attachments/assets/10075903-5785-458b-8a51-3f5a24accf88)
![image](https://github.com/user-attachments/assets/bfea2bea-37ff-488f-92e6-d561e9b98b8d)
It's really bloated😕

**Comment by ajsglist** [comment]
ultimately it's just a minor cosmetic feature, so it's something that I don't really have a problem foregoing.  Given my experiences with similarly large mods with unaddressed performance issues, the mod you suggested is something I'm going to continue to avoid.

But again, thanks for the suggestion.  And I hope it works well for you :)
